docker-compose.yml

```diff
@@ -93,6 +93,7 @@ services:
       - ./public:/usr/src/code/public
       - ./src:/usr/src/code/src
       - ./dev:/usr/local/dev
+      - ./vendor/utopia-php/database:/usr/src/code/vendor/utopia-php/database
```

Comment:
Please avoid committing this changes, they make tests break and are very hard to detect.